pkg/skaffold/build/cluster/cluster.go

```diff
@@ -82,7 +82,7 @@ func (b *Builder) runBuildForArtifact(ctx context.Context, out io.Writer, a *lat
 		return b.buildWithKaniko(ctx, out, a.Workspace, a.ImageName, a.KanikoArtifact, tag, requiredImages)
 
 	case a.CustomArtifact != nil:
-		return custom.NewArtifactBuilder(nil, b.cfg, true, append(b.retrieveExtraEnv(), util.EnvPtrMapToSlice(requiredImages, "=")...)).Build(ctx, out, a, tag)
+		return custom.NewArtifactBuilder(nil, b.cfg, true, false, append(b.retrieveExtraEnv(), util.EnvPtrMapToSlice(requiredImages, "=")...)).Build(ctx, out, a, tag)
```

Comment:
instead of `false` always read from `b.cfg.SkipTests()`. You'll have to add the method to the `Config` interface but it should already be implemented by `b.cfg`.